app/containers/SignUp/actions.js

```diff
@@ -36,3 +46,24 @@ export function setReducerDefault() {
     type: SET_REDUCER_DEFAULT,
   };
 }
+
+/*
+ *
+ * signUp modal window actions
+ *
+ */
+
+export function showSignUpModal(content) {
+  window.$(`#${SIGN_UP_MODAL_ID}`).modal('show');
```

Comment:
hiding and showing should be the function of the modal. ModalComponent needs to expose prop to components that are using it. E.g
```
<ModalComponent show={show} > ...
```